.ci.yaml

```diff
@@ -12,12 +12,15 @@ platform_properties:
   linux:
     properties:
       os: Linux
+      bot_type: "regular"
```

Comment:
Revisiting this, can we make the property more descriptive? Regular is vague. Maybe we could do something like:

signing_certs: None

Then we can add this in flutter/infra at the builder template

Reply:
Good advice! I will update the robot configs to use this as well